aamp/parameters.py

```diff
@@ -77,42 +79,34 @@ def __init__(self, type_: str='xml', version=0) -> None:
     def __repr__(self) -> str:
         return f'ParameterIO(type_={self.type}, version={self.version}, lists={repr(self.lists)})'
 
+@dataclass
 class Vec2:
     __slots__ = ['x', 'y']
-    def __init__(self, x=0.0, y=0.0) -> None:
-        self.x = x
-        self.y = y
-    def __repr__(self) -> str:
-        return f'Vec2(x={self.x}, y={self.y})'
+    x: float
```

Comment:
```suggestion
    x: float = 0.0
```
and same for the other attributes below